### ISSUE TITLE
Feature-Request: user can change API URL

### ISSUE THREAD

**Issue by GoogleCodeExporter** [gyc2432968/python-twitter#128]
```
i mean not only allow to use twitter.com as API url.

in China, twitter.com is not forbiden to visit, but there are api proxys, 
we can use api changable clients to contact twitter.

hope python-twitter can support this fuction
```

Original issue reported on code.google.com by `jarry...@gmail.com` on 7 Mar 2010 at 5:17
- Merged into: #37

**Comment by GoogleCodeExporter** [comment]
```
I'm attaching the patch(es) for this enhancement.

Tested it using http://nest.onedd.net/api
```

Original comment by `patricio...@gmail.com` on 17 Apr 2010 at 8:47

Attachments:
- [twitter.py.patch](https://storage.googleapis.com/google-code-attachments/python-twitter/issue-128/comment-1/twitter.py.patch)
- [twitter_test.py.patch](https://storage.googleapis.com/google-code-attachments/python-twitter/issue-128/comment-1/twitter_test.py.patch)

**Comment by GoogleCodeExporter** [comment]
```
This was fixed when Issue 37 was fixed as part of revision dca94e8bd2

A base_url parameter was added to the API init call
```

Original comment by `bear42` on 16 Oct 2010 at 8:04
- Changed state: **Duplicate**
